src/components/carsoulSection/carsoureItem.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import {
+  MDBCard,
+  MDBCardBody,
+  MDBCardText,
+  MDBCarouselItem,
+  MDBCol,
+  MDBRow,
+  MDBContainer,
+} from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+
+// create 4 items for the carsouel movement
+function CarouselItems() {
+  const { t } = useTranslation();
+  let cards = [];
+  
+  let booksArray = [
+    {
+      cover:
+        'http://1.bp.blogspot.com/-yQklGusIOBE/Ul-oQ0_9vOI/AAAAAAAAfI4/XQTmNLaJZLQ/s1600/book-cover-poster-27.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22,333',
+      id: 1,
+    },
+    {
+      cover:
+        'http://www.snugglyoranges.com/wp-content/uploads/2014/09/Unspoken.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 2,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 3,
+    },
+    {
+      cover:
+        'https://th.bing.com/th/id/OIP.Zf2SkeIiSI4ymvT2cc4u2AHaKd?pid=Api&rs=1',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 4,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 5,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 6,
+    },
+  ];
+
+  let createCards = () => {
+    let specialBooks = [];
```

Comment:
please change this into `const`
when ever you can use `const` please don't use let